app/objects/player.py

```diff
@@ -578,12 +575,11 @@ async def unsilence(self, admin: "Player") -> None:
             {"silence_end": self.silence_end, "user_id": self.id},
         )
 
-        log_msg = f"{admin} unsilenced."
         await app.state.services.database.execute(
             "INSERT INTO logs "
-            "(`from`, `to`, `msg`, `time`) "
-            "VALUES (:from, :to, :msg, NOW())",
-            {"from": admin.id, "to": self.id, "msg": log_msg},
+            "(`from`, `to`, `action`, `msg`, `time`) "
+            "VALUES (:from, :to, 'unsilence', NULL, NOW())",
+            {"from": admin.id, "to": self.id},
```

Comment:
''

Reply:
solved in https://github.com/cmyui/gulag/commit/6710da3c927f7afdf628e4f7d74d97314d27adc3